### PR TITLE
all: Gradle project structure simplification

### PR DESCRIPTION
**Pull request by lijamie98**
<details>
  <summary>PR Checklist</summary>

### PR Structure

* [ ] This PR has reasonably narrow scope (if not, break it down into smaller PRs).
* [ ] This PR avoids mixing refactoring changes with feature changes (split into two PRs
  otherwise).
* [ ] This PR's title starts with name of package that is most changed in the PR, ex.
  `paymentservice.stellar`, or `all` or `doc` if the changes are broad or impact many
  packages.

### Thoroughness

* [ ] This PR adds tests for the most critical parts of the new functionality or fixes.
</details>

### What

1. Removed `config-spring-property`
2. Removed `data-spring-jdbc`
3. Created `service-runner` for the ServiceRunner class and `anchor-platform-runner.jar`
4. Created `integration-tests` that runs only integration tests.
5. Clean up unused dependencies.

### Why

To simplify the project structure.

### Known limitations

`api-schema` will be created in the next PR.

### CODE REVIEW

File: platform/src/test/resources/test-anchor-config.yaml [hunk 1]
```diff
@@ -27,7 +27,6 @@ app-config:
     jwtSecretKey: secret
 
   # sep-1
-  sep1:
```
Comment (marcelosalloum) [inline platform/src/test/resources/test-anchor-config.yaml:30]: ❗  Wait, I think you removed this by mistake?

**Review by marcelosalloum — COMMENTED** [review]
Looking great! I added 2 comments with ❗ that should be fixed before merging. Also, for some reason the build is not working anymore, we should investigate it.

**Comment by marcelosalloum** [comment]
Important links about the build that's failing:
* [Buildmeister jobs](https://buildmeister-v3.stellar-ops.com/job/ProductEngineering/job/stellar-anchor-platform-preview/)
* [Jenkins command](https://github.com/stellar/pipelines/blob/af489745d8cd2887f5ab308629dc2a67e09fe4ea/anchor-platform/Jenkinsfile-preview#L97-L101)

File: README.md [hunk 2]
```diff
@@ -8,10 +8,11 @@
 The Stellar Anchor SDK is a collection of projects that make easy to build financial applications on Stellar.
 
 * [core](./core): the base package for implementing standardized anchor applications.
-* [config-spring-property](./config-spring-property): Implementation of configuration management with [Spring ConfigurationProperties](https://docs.spring.io/spring-boot/docs/current/reference/html/features.html#features.external-config).
-* [data-spring-jdbc](./data-spring-jdbc): Implementation of data access with [Spring Data JDBC](https://spring.io/projects/spring-data-jdbc). 
-* [payment-circle](./payment-circle): Implementation of payment servie with [Circle API](https://developers.circle.com/reference).
-* [platform](./platform): The anchor platform [Spring Boot Application with WebMVC](https://spring.io/guides/gs/serving-web-content/). 
+* [payment-circle](./payment-circle): implementation of payment servie with [Circle API](https://developers.circle.com/reference).
```
[the remark this answers is not in the document]
Comment (lijamie98) [reply README.md:11]: Thanks for catching this!

File: README.md [hunk 3]
```diff
@@ -8,10 +8,11 @@
...
+* [platform-apis](./platform-apis): the API interfaces, request, response classes for the anchor platform.
```
[the remark this answers is not in the document]
Comment (lijamie98) [reply README.md:12]: Yes, that is right.

File: integration-tests/src/test/resources/test-anchor-config.yaml [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+    enabled: true
+    kafkaBootstrapServer: localhost:29092
+    useSingleQueue: false
+    eventTypeToQueue:
+      all: ap_event_single_queue
+      quote_created: ap_event_quote_created
+      transaction_created: ap_event_transaction_created
+      transaction_payment_received: ap_event_transaction_payment_received
+      transaction_error: ap_event_transaction_error
+
```
[the remark this answers is not in the document]
Comment (lijamie98) [reply integration-tests/src/test/resources/test-anchor-config.yaml:96]: This should be good.

File: platform/src/test/resources/test-anchor-config.yaml (hunk 1 above)
Comment (lijamie98) [reply platform/src/test/resources/test-anchor-config.yaml:30]: Fixed.

**Review by marcelosalloum — APPROVED** [review]

**Pull request merged**